### DISCUSSION TITLE
Arcsi letöltési/hallgatási számok

### DISCUSSION THREAD

**Discussion by gammaw** [lahmacunradio/liquidsoap#396] [Ideas (analytics)]
Megmutatás (frontend)

**Comment by pvj** [comment]
Átvittem megfelelő új honlapos helyre

**Reply by gammaw** [reply]
Cool! Azon agyalgattam, hogy töröljük-e itt akkor, és arra jutottam, hogy attól, hogy pl. az új honlap mondjuk nem valósítja meg (mert úgy dönt a csapat), a (standalone) arcsi projektnek attól még ez lehet egy issue. Persze ez egyelőre inkább elméleti lehetőség, szóval az is lehet, hogy jobb ha nem tartunk ilyen duplikátumokat. Szerinted? Te is ezért tartottad meg?

**Comment by pvj** [comment]
Hat igen, meg ezt meg lehet eleg sokat beszelgetni hogy mit mutassunk /
hogyan / hol stb.

On Thu, Jun 3, 2021 at 6:49 PM gammaw ***@***.***> wrote:

> Cool! Azon agyalgattam, hogy töröljük-e itt akkor, és arra jutottam, hogy
> attól, hogy pl. az új honlap mondjuk nem valósítja meg (mert úgy dönt a
> csapat), a (standalone) arcsi projektnek attól még ez lehet egy issue.
> Persze ez egyelőre inkább elméleti lehetőség, szóval az is lehet, hogy jobb
> ha nem tartunk ilyen duplikátumokat. Szerinted? Te is ezért tartottad meg?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mmmnmnm/lahmacun/discussions/396#discussioncomment-821824>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAUE676CIKUUBCSJVCW47DLTQ6XDNANCNFSM42JKSJNQ>
> .
>


-- 
^\^
